deepspeed/runtime/zero/config.py

```diff
@@ -41,6 +41,7 @@
     "zero_quantized_gradients": [true|false],
     "memory_efficient_linear": [true|false],
     "override_module_apply": [true|false],
+    "graph_harvesting": [true|false],
```

Comment:
Please move `graph_harvesting` outside `zero` to the global namespace, since it can be useful for other parts of deepspeed.

Reply:
@inkcherry, can you please help with this change?

Reply:
moved.